app/src/main/java/com/shagalalab/constitution/ui/article/ArticleViewModel.kt

```diff
@@ -5,35 +5,39 @@ import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.shagalalab.constitution.data.dao.ArticleDao
 import com.shagalalab.constitution.data.models.ArticleModel
-import java.util.concurrent.Executors
+import java.util.concurrent.Executor
 
-class ArticleViewModel(private val articleDao: ArticleDao) : ViewModel() {
+class ArticleViewModel(
+    private val articleDao: ArticleDao,
+    private val executorService: Executor
```

Comment:
Can you please rename `executorService` to `executor` here and in all other places?